harness/testTypedArrays.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2014 Domenic Denicola. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
```

Comment:
Indeed, sorry for the confusion there, I was scanning through all of the files quickly. I'll update my main comment accordingly.